docs/source/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import git_machete
```

Comment:
Actually! how does Sphinx know to look into this directory when building the docs?

Reply:
Or maybe more specifically, what are the exact commands it's running to build the repository? is it `tox -e docs` somehow?

Reply:
(is it configurable in any way?)

Reply:
It will be easier while doing #210, but to copy some of the stuff from the rtd:
![Screenshot 2021-09-03 at 14 35 42](https://user-images.githubusercontent.com/13608913/132006092-fe418721-0fc0-4534-adf1-31fe1d9825ce.png)
![Screenshot 2021-09-03 at 14 35 52](https://user-images.githubusercontent.com/13608913/132006122-018399b5-e917-4719-8d37-dbb2e3446084.png)

Commands:

- git clone --no-single-branch --depth 50 https://github.com/VirtusLab/git-machete .
- git checkout --force origin/docs/read_the_docs
- git clean -d -f -f
- python3.7 -mvirtualenv /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m pip install --upgrade --no-cache-dir pip setuptools
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m pip install --upgrade --no-cache-dir mock==1.0.1 pillow==5.4.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx sphinx-rtd-theme readthedocs-sphinx-ext<2.2
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m pip install --exists-action=w --no-cache-dir -r requirements.txt
- home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python /home/docs/checkouts/readthedocs.org/user_builds/git-machete/checkouts/latest/setup.py install --force
- cat docs/source/conf.py
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m sphinx -T -b html -d _build/doctrees -D language=en . _build/html
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m sphinx -T -b readthedocssinglehtmllocalmedia -d _build/doctrees -D language=en . _build/localmedia
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m sphinx -b latex -D language=en -d _build/doctrees . _build/latex
- cat latexmkrc
- latexmk -r latexmkrc -pdf -f -dvi- -ps- -jobname=git-machete -interaction=nonstopmode
- mv -f /home/docs/checkouts/readthedocs.org/user_builds/git-machete/checkouts/latest/docs/source/_build/latex/git-machete.pdf /home/docs/checkouts/readthedocs.org/user_builds/git-machete/artifacts/latest/sphinx_pdf/git-machete.pdf
- /home/docs/checkouts/readthedocs.org/user_builds/git-machete/envs/latest/bin/python -m sphinx -T -b epub -d _build/doctrees -D language=en . _build/epub
- mv -f /home/docs/checkouts/readthedocs.org/user_builds/git-machete/checkouts/latest/docs/source/_build/epub/git-machete.epub /home/docs/checkouts/readthedocs.org/user_builds/git-machete/artifacts/latest/sphinx_epub/git-machete.epub